docs/getting_started/quickstart.md

```diff
@@ -20,9 +20,19 @@ export PROMPTGUARD_API_KEY="..."
 PromptGuard offers two main functions: `sanitize()` and `desanitize()`. `sanitize()` takes a string and returns a sanitized (i.e. encrypted and redacted) version of it, while `desanitize()` takes a sanitized string and returns the original string.
 
 ```python
+import promptguard as pg
+
+input_text = "Some string with sensitive PII"
```

Comment:
Perhaps include actual simulated PII here? That way, if someone runs this as-is, they'll see a redaction in the result.